src/azure-cli/azure/cli/command_modules/configure/tests/latest/test_configure.py

```diff
@@ -55,5 +55,15 @@ def test_configure_global_defaults(self):
         self.assertTrue(actual == expected)
 
 
+class LocalContextCommandsScenarioTest(LocalContextScenarioTest):
+
+    def test_local_context_commands(self):
+        self.cmd('local-context list')
+        self.cmd('local-context show --scope ALL')
```

Comment:
Let's make `ALL` case-insensitive to follow the Azure CLI convention.

Reply:
Updated.